docker-compose.dev.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
```

Comment:
I understand that you chose the naming of the compose file to support newcomers with docker to start everything in prod mode. However, i would rename it the other way around, anologe to the env files: docker-compose.yml => dev, docker-compose.prod.yml => prod (consistency)

Reply:
Done